tests/Feature/Http/Controllers/Auth/EmailVerificationTest.php

```diff
@@ -30,7 +30,7 @@ protected function setUp(): void
     public function testEmailVerificationScreenCanBeRendered(): void
     {
         // Given
-        $user = User::factory()->create([
+        $user = AuthorizedUser::factory()->create([
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create([
````